cmd/contour/servecontext.go

```diff
@@ -63,7 +66,9 @@ type serveContext struct {
 	ingressClass string
 
 	// Address to be placed in status.loadbalancer field of Ingress objects.
-	ingressStatusAddress string
+	// May be either a literal IP address or another string.
```

Comment:
```suggestion
	// May be either a literal IP address or a host name.
```

Reply:
I didn't specify hostname because we don't check for it, but it's probably more correct.